### DISCUSSION TITLE
Tips: Show user/group of files in status bar dpoesnt work

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#563] [1. Q&A] [answered]
I try "Show user/group of files in status bar" code, but doesnt work. I get error message:
"thread 'main' panicked at yazi-plugin/src/plugin.rs:42:32:
called `Result::unwrap()` on an `Err` value: SyntaxError { message: "[string \"yazi-plugin/src/plugin.rs:42:17\"]:121: malformed number near '0.2.1'", incomplete_input: false }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Aborted"

I use the following code in init.lua:

```
function Status:owner()
	local h = cx.active.current.hovered
	if h == nil or ya.target_family() ~= "unix" then
		return ui.Line {}
	end

	return ui.Line {
		ui.Span(ya.user_name(h.cha.uid) or tostring(h.cha.uid)):fg("magenta"),
		ui.Span(":"),
		ui.Span(ya.group_name(h.cha.gid) or tostring(h.cha.gid)):fg("magenta"),
		ui.Span(" "),
	}
end

function Status:render(area)
	self.area = area

	local left = ui.Line { self:mode(), self:size(), self:name() }
    local right = ui.Line { self:owner(), self:permissions(), self:percentage(), self:position() }
	return {
		ui.Paragraph(area, { left }),
		ui.Paragraph(area, { right }):align(ui.Paragraph.RIGHT),
		table.unpack(Progress:render(area, right:width())),
	}
end
```

**Comment by sxyazi** [comment, marked as answer]
it works for me by using your code, yazi 0.2.2 (56ede51 2024-01-22)

![1](https://github.com/sxyazi/yazi/assets/17523360/2776ae88-0589-4769-b08e-da5c24cea809)

**Comment by og900aero** [comment]
Oh, I left a ' sign in it.
